### PR TITLE
Add AtomOne chain

### PR DESCRIPTION
**Pull request by tbruyelle**

### CODE REVIEW

File: atomone/chain.json [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+      "tx_page": "https://explorer.allinbits.com/atomone/tx/${txHash}",
+      "account_page": "https://explorer.allinbits.com/atomone/account/${accountAddress}"
+    }
+  ],
+  "images": [
+    {
+      "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/atomone/images/atomone.png",
+      "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/atomone/images/atomone.svg",
+      "theme": {
+        "primary_color_hex": "#65d6d4"
```
Comment (JeremyParish69) [inline atomone/chain.json:137]: How was this color selected?

File: atomone/chain.json [hunk 2]
```diff
@@ -0,0 +1,141 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "atomone",
+  "chain_type": "cosmos",
+  "chain_id": "atomone-1",
+  "website": "https://atom.one",
+  "pretty_name": "AtomOne",
+  "status": "live",
+  "network_type": "mainnet",
+  "bech32_prefix": "atone",
```
Comment (JeremyParish69) [inline atomone/chain.json:10]: Please register this prefix:
https://github.com/satoshilabs/slips/blob/master/slip-0173.md

Comment (tbruyelle) [reply atomone/chain.json:10]: It's already registered https://github.com/satoshilabs/slips/pull/1803

File: atomone/chain.json (hunk 1 above)
Comment (tbruyelle) [reply atomone/chain.json:137]: To be honest, I tried to find out where this color is used in cosmos.directory and couldn't find any place where it was used. So I concluded that it didn't matter and left it unchanged.

File: atomone/chain.json (hunk 2 above)
Comment (tbruyelle) [reply atomone/chain.json:10]: oh damned there's a typo, let me fix that.

File: atomone/chain.json (hunk 1 above)
Comment (JeremyParish69) [reply atomone/chain.json:137]: Ok. Well if you aren't going to select the correct color, then please omit the property

Comment (tbruyelle) [reply atomone/chain.json]: Done here 9f0c24e

File: atomone/assetlist.json [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/atomone/images/atomone.png",
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/atomone/images/atomone.svg"
+      },
+      "images": [
+        {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/atomone/images/atomone.png",
+          "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/atomone/images/atomone.svg",
+          "theme": {
+            "primary_color_hex": "#65d6d4"
```
Comment (JeremyParish69) [inline atomone/assetlist.json:30]: There is still an incorrect color here

**Pull request merged**